docs/section5/Rule5-7-1.md

```diff
@@ -2,7 +2,7 @@
 **Rule ID:** 5-7-1  
 **Rule Description:** Baseline WWR should be equal to User WWR or 40%, whichever is smaller.  
```

Comment:
@xingcx this rule description is incorrect, apologies for that. This used to be the requirement in App G 90.1-2010 but the new App G requires vertical fenestration areas for new buildings and additions shall equal that in Table G3.1.1-1
based on the area of gross above-grade walls that separate conditioned spaces and semiheated spaces from the exterior.

Reply:
Updated.